src/app/cube/details/details.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="!loading.length; else loadingTemplate">
   <div *ngIf="learningObject && !errorStatus && revisedLearningObject" class="details-wrapper" [ngClass]="learningObject.length">
-    <cube-details-splash [canRate]="loggedin && auth.user.emailVerified && !isOwnObject" [ratings]="ratings" [averageRating]="averageRatingValue" [learningObject]="learningObject" (showNewRating)="showAddRating = $event;"></cube-details-splash>
+    <cube-details-splash [canRate]="loggedin && auth.user.emailVerified && !isOwnObject" [canViewChangelogs]="loggedin && auth.user.emailVerified" [ratings]="ratings" [averageRating]="averageRatingValue" [learningObject]="learningObject" (showNewRating)="showAddRating = $event;" (openChangelogModal)="openViewAllChangelogsModal()"></cube-details-splash>
```

Comment:
Is email verification required to view changelogs? If so, the backend business logic will need to be updated. Does anyone have context on this? @Cyber4All/front-end @DonSeannelly

Reply:
I guess it wouldn't need to be in the sense that the user isn't leaving anything on the site (such as a rating), and is just viewing the history of a learning object. But I am open to hear other opinions on this.

Reply:
Email verification is required for Learning Object ratings to prevent potential bot spam.